xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/RippledServerInfo.java

```diff
@@ -0,0 +1,243 @@
+package org.xrpl.xrpl4j.model.client.serverinfo;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.annotations.Beta;
+import com.google.common.primitives.UnsignedInteger;
+import com.google.common.primitives.UnsignedLong;
+import org.immutables.value.Value;
+
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+/**
+ * An implementation of {@link ServerInfo} that conforms to rippled server payloads.
+ */
+@Beta
+@Value.Immutable
+@JsonSerialize(as = ImmutableRippledServerInfo.class)
+@JsonDeserialize(as = ImmutableRippledServerInfo.class)
+public interface RippledServerInfo extends ServerInfo {
+
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableRippledServerInfo.Builder}.
+   */
+  static ImmutableRippledServerInfo.Builder builder() {
+    return ImmutableRippledServerInfo.builder();
+  }
+
+  default ServerInfoType type() {
+    return ServerInfoType.RIPPLED_SERVER_INFO;
+  }
+
+  /**
+   * If {@code true}, this server is amendment blocked.
+   *
+   * @return {@code true} if this server is amendment blocked, otherwise {@code false}.
+   */
+  @JsonProperty("amendment_blocked")
+  @Value.Default
+  default boolean amendmentBlocked() {
+    return false;
+  }
+
+  /**
+   * The version number of the running rippled version.
+   *
+   * @return A {@link String} containing the version number.
+   */
+  @JsonProperty("build_version")
+  String buildVersion();
+
+  /**
+   * On an admin request, returns the hostname of the server running the rippled instance; otherwise, returns a single
+   * RFC-1751 word based on the node public key.
+   *
+   * @return A {@link String} containing the host name of the server.
+   */
+  @JsonProperty("hostid")
+  String hostId();
+
+  /**
+   * Amount of time spent waiting for I/O operations, in milliseconds. If this number is not very, very low, then the
+   * rippled server is probably having serious load issues.
+   *
+   * @return An {@link UnsignedLong} representing the I/O latency.
+   */
+  @JsonProperty("io_latency_ms")
+  UnsignedLong ioLatencyMs();
+
+  /**
+   * The number of times (since starting up) that this server has had over 250 transactions waiting to be processed at
+   * once. A large number here may mean that your server is unable to handle the transaction load of the XRP Ledger
+   * network. For detailed recommendations of future-proof server specifications, see
+   * <a href="https://xrpl.org/capacity-planning.html">Capacity Planning</a>.
+   *
+   * @return An optionally-present {@link String}.
+   */
+  @JsonProperty("jq_trans_overflow")
+  Optional<String> jqTransOverflow();
```

Comment:
I think this field is already in `ServerInfo`. Do we need it here as well?

Reply:
Should be gone from ServerInfo. Only ever exists in RippledServerInfo